build.gradle

```diff
@@ -134,8 +134,6 @@ dependencies {
 
   compile group: 'uk.gov.service.notify', name: 'notifications-java-client', version: '3.8.0-RELEASE'
 
-  compile group: 'uk.gov.hmcts.reform', name: 'java-logging-spring', version: versions.reformJavaLogging
-  compile group: 'uk.gov.hmcts.reform', name: 'java-logging-httpcomponents', version: versions.reformJavaLogging
```

Comment:
by removing these components, `java-logging` one is not included. if that's intentional - all good

Reply:
I compared the logging output and it looked the same to me, 
Wasn't really sure if I should add the `java-logging` dep here.
To be honest java-logging isn't doing much now, for us its just providing the app insights filters etc which won't be needed when the appinsights spring boot starter is released in the next month or two hopefully

Reply:
Turns out the dep was already there but it was version 1.4.0 coming from the pdf service client, I've added the dep explicitly here to keep them in sync